### ISSUE TITLE
To: input field reccomendations almost unusable

### ISSUE THREAD

**Issue by mart-ffbsee** [the-djmaze/snappymail#769]
**SnappyMail version, browser, browser version:**
2.23.0 nextcloud  *(edited because of earlier typo)*

**Expected behavior and actual behavior:**
Expected: 
If you type in the first few letters of a name in the to: input field, a dropdown with names should appear (and stay visible)

Actual: 
When you type in the first letter the dropdown appears (but out of place) but vanishes as soon as the second letter is typed. Therefore it is next to impossible to find some entries that would pop up when more letters were typed (and if they would be searched for)

**Steps to reproduce the problem:**
Type one letter in the To: input field, observe. Then type in second letter.

**Logs or screenshots:**

**Comment by the-djmaze** [comment]
Did you read issues #368 and #513 ?

**Comment by mart-ffbsee** [comment]
Thanx for your reply.
My issue is not the same as in the two mentioned issues as far as can see.
I'm am using Chromium, but it also happens in Firefox.

My issue is: As soon as you type a second letter into the field the whole  list with the suggestions disappears completely. If I then delete the second letter that I typed, the list re-appears again.

BUT
I just found out how to get rid of that problem:
I had the extension "Contact suggestions (IMAP folder)" active.

As soon as I disable said extension the problem is gone.

So, it seems to me that the problem is related to the extension "Contact suggestions (IMAP folder)".
